pkg/ip/link.go

```diff
@@ -41,6 +41,15 @@ func makeVethPair(name, peer string, mtu int) (netlink.Link, error) {
 	return veth, nil
 }
 
+func peerExists(name string) bool {
+
+	_, err := netlink.LinkByName(name)
+	if err != nil {
```

Comment:
If any returned values (_, err) isn't used out of "if" block - this should be combined into:
```
  if _, err := netlink.LinkByName(name); err != nil {
```
Like shown in second example in https://golang.org/doc/effective_go.html#if